modules/broker-core/src/main/grammar/SelectorParser.jj

```diff
@@ -91,7 +94,7 @@ TOKEN [IGNORE_CASE] :
 
 TOKEN [IGNORE_CASE] :
 {
-    <NUMERIC_LITERAL: (["0"-"9"])+>
+    <NUMERIC_LITERAL: (["0"-"9"])+(["L"])*>
```

Comment:
Why have we added `(["L"])*` here? Won't that match strings like `39393LLLLLLLLL`?

Reply:
Yes. Then that will be treated as a Long value internally